src/main/kotlin/com/atlassian/performance/tools/report/api/judge/ShiftedDistributionRegressionTestProvider.kt

```diff
@@ -0,0 +1,19 @@
+package com.atlassian.performance.tools.report.api.judge
+
+import com.atlassian.performance.tools.report.api.ShiftedDistributionRegressionTest
+
+open class ShiftedDistributionRegressionTestProvider {
+    open fun get(
+        baseline: DoubleArray,
+        experiment: DoubleArray,
+        mwAlpha: Double,
+        ksAlpha: Double
+    ): ShiftedDistributionRegressionTest {
+        return ShiftedDistributionRegressionTest(
+            baseline = baseline,
+            experiment = experiment,
+            ksAlpha = ksAlpha,
+            mwAlpha = mwAlpha
+        )
+    }
```

Comment:
Quite an anemic API. What is it needed for?

Reply:
I wish we could discuss the topic of anemic API desing. I personally don't understand the idea of what is is and why it's an antipattern despite of reading the https://www.martinfowler.com/bliki/AnemicDomainModel.html (apparently it's origin?)

Reply:
I'm in the office tomorrow, we can discuss it live.
If we reach consensus we can document the decision, e.g. in [go/jpt](https://go.atlassian.com/jpt).
If you wish to record the whole discussion, we can do it too, maybe by resurrecting the [old JPT Slack](go.atlassian.com/jpt-slack), or maybe by GitHub Pages?

Reply:
I think there is a difference between anemic model and provider - which is used
- not to use a constructor in other class
- make dependency injectable and testable